ocaml/xapi/xapi_xenops.ml

```diff
@@ -2023,6 +2024,11 @@ let on_xapi_restart ~__context =
 		Db.VM.set_resident_on ~__context ~self:vm ~value:Ref.null;
 	) xapi_vms_not_in_xenopsd
 
+  info "applying guest agent configuration during restart";
```

Comment:
I doubt this compiles because you need a semicolon before this sequence.

Reply:
Yes, updated.